include/rmm/error.hpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
Shouldn't the copyright still be 2020-2024 -- this is old code.

```suggestion
 * Copyright (c) 2020-2024, NVIDIA CORPORATION.
```